content/cpp/concepts/vectors/vectors.md

```diff
@@ -42,9 +42,9 @@ std::vector<double> order = {3.99, 12.99, 2.49};
 
 ## Index
 
-An index refers to an element’s position within an ordered list, like a vector or an array. The first element has an index of 0.
+An index refers to an element’s position within an ordered list, like a `vector` or an array. The first element has an index of 0.
```

Comment:
For this specific line I would make this suggestion since this line mentions arrays for the first time:

```suggestion
An index refers to an element’s position within an ordered list, like a vector or an [array](https://www.codecademy.com/resources/docs/cpp/arrays). The first element has an index of 0.
```